### PR TITLE
[Merged by Bors] - feat: port AlgebraicGeometry.Morphisms.Basic

### PR DESCRIPTION
**Pull request by jjaassoonn**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/Morphisms/Basic.lean [hunk 1]
```diff
@@ -11,8 +11,9 @@ Authors: Andrew Yang
 import Mathlib.AlgebraicGeometry.AffineScheme
 import Mathlib.AlgebraicGeometry.Pullbacks
 import Mathlib.CategoryTheory.MorphismProperty
+import Mathlib.Data.List.TFAE
 
-/-!
+/-!  h y y y y y y j
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/Morphisms/Basic.lean:16]: I suppose this is not intentional?
```suggestion
/-!
```

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/AlgebraicGeometry/Morphisms/Basic.lean (hunk 1 above)
Comment (jjaassoonn) [reply Mathlib/AlgebraicGeometry/Morphisms/Basic.lean]: Indeed, don't know how it got there

**Pull request closed without merging**